src/lib.rs

```diff
@@ -44,6 +45,7 @@ impl Router {
             module_cache_dir: PathBuf::default(),
             base_log_dir: PathBuf::default(),
             default_host: String::default(),
+            http_concurrency: Option::None,
```

Comment:
`None` is imported by default in rust, so this can just be `http_concurrency: None`

Reply:
done